### PR TITLE
Send ETH to Smart Contract

### PR DESCRIPTION
**Pull request by ErvinDimitri**
The issue was in getting the correct Gas Limit, and for that, the amount cant be 0
Each time, the value input changed, it was sending the value to estimateFormFees() to get the Fee Data. 
For example, when an user is typing 0.005

### CODE REVIEW

**Comment by ErvinDimitri** [comment]
Hello
Just fixed the issue in sending ETH to a smart contract
Let me know, if everything is alright

File: src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx [hunk 1]
```diff
@@ -91,6 +91,10 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
 
     if (!wallet) throw new Error('No wallet connected')
 
+    if(Number(values.cryptoAmount) === 0){
+      values.cryptoAmount="0.00001"
+    }
```
Comment (mrnerdhair) [inline src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx:96]: I don't think this is a practical solution. It's going to make everything inaccurate, and isn't guaranteed to make the fee estimation correct. A better solution (though still not good) would be adding a fixed fudge-factor to the gasLimit.

**Review by mrnerdhair — CHANGES_REQUESTED** [review]

**Comment by mrnerdhair** [comment]
I broke out the 1559 fee support part here into #1181 (issue) and #1182 (pr).

File: src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx (hunk 1 above)
Comment (0xdef1cafe) [reply src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx:96]: yeah sorry @ErvinDimitri this is not a viable solution

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

File: src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx (hunk 1 above)
Comment (ErvinDimitri) [reply src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx:96]: I performed some tests and concluded that `getFeeData()` is working as expected.
Here is a different approach, to estimate a Gas Limit, only when the input value is greater than 0
Please check the PR: https://github.com/shapeshift/web/pull/1215

**Comment by ErvinDimitri** [comment]
> @ErvinDimitri since you opened a new PR with a different solution, could you close this one? Thank you!

yes, sure

**Pull request closed without merging**